src/ensembl/src/content/app/browser/Browser.tsx

```diff
@@ -102,6 +102,7 @@ export const Browser: FunctionComponent<BrowserProps> = (
     TrackStates
   >({});
   const lastGenomeIdRef = useRef(props.activeGenomeId);
+  const drawerOpenedForGenome = props.drawerOpened[props.activeGenomeId];
```

Comment:
This belongs in the selector, not in the component itself. The component shouldn't care about genomes other than active.